public/assets/js/scripts.js

```diff
@@ -285,4 +285,27 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+  $('.search-calculate-button').click(function(){
+    let periodic = $('.periodic-investment').val();
+    let initial = $('.initial-investment').val();
+    let maxValue = removeCommasAndMakeNumber($('.investment-value-input').data('max-amount'));
+    let frequency = $("input[name='investment-frequency']:checked").val();
+  
+    $.ajax({
+      type: 'POST',
+      url: './includes/calculate-recommendation.php',
```

Comment:
Using a single method for the whole recommendation makes it very difficult. Please create different methods for each recommendation and check if the user input satisfies the condition in the method.

Here is my thought process:
1. Get all user inputted values (periodic, initial and frequency)
2. Calculate the number of months needed to complete payments `monthsToCompletePayment` from the values above. monthsToCompletePayment can be calculated with `balance/periodic`.
3. Write different functions for each category that accept the amount, initial payment and monthsToCompletePayment. E.g
  a. recommendPersonal
  b. recommendMortgage
  c. recommendRentToOwn e.t.c.
 ...
Here is a sample code for recommendPersonal and Spread Payments
```js
// written exactly like the docs - Initial Payments >= 50%, completed <= 6 months
const recommendPersonal = (amount, initial, monthsToCompletePayment) => (initial >= amount * 0.5) && monthsToCompletePayment <= 6;

const recommendSpread = (amount, initial, monthsToCompletePayment) => (initial >= amount * 0.2) && monthsToCompletePayment > 6 && monthsToCompletePayment <= 24;
```
4. Use a conditional to call each of the functions above to be sure if the recommendation will be made.
```js
if (recommendPersonalCategory(amount, initial, monthsToCompletePayment)) {
// the choice is yours here, you can assign to a variable here or assign to an array here and loop over the array to get recommended choices, 
}

if (recommendMortgage(amount, initial, monthsToCompletePayment)) {
// the choice is yours here, you can assign to a variable here or assign to an array here and loop over the array to get recommended choices, 
}
```
5. Place the recommended choices back in the search result page on the `Awesome Section`.